int32[] posesId
geometry_msgs/Pose[] poses
